parachain/src/wasm_executor/validation_host.rs

```diff
@@ -40,7 +43,7 @@ enum Event {
 }
 
 lazy_static::lazy_static! {
-	pub static ref HOST: Mutex<ValidationHost> = Mutex::new(ValidationHost::new());
+	static ref HOSTS: Mutex<HashMap<ParaId, Arc<Mutex<ValidationHost>>>> = Mutex::new(Default::default());
```

Comment:
why organize it by para-id? i think it's more likely that we would run jobs of the same parachain at the same time than to run jobs of different parachains at the same time.

Reply:
And couldn't we drop the `Arc`?

Reply:
The `Arc` is there so that each `ValidationHost` can be locked outside of the `HOSTS` mutex